packages/story/src/stories/todo/todo.stories.tsx

```diff
@@ -1,7 +1,8 @@
 import type { Meta, StoryObj } from "@storybook/react";
 import Component from "../../lib/Notion";
 import json from "./todo.json";
-import { Todo, TodoArgs } from "@notionpresso/react";
+import { Todo } from "../../../../core/src/lib";
+import { TodoArgs } from "../../../../core/src/lib";
```

Comment:
p1: don't import with relative path

it is mono repo

see: workspace property on root package.json